infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/LogReplicationContext.java

```diff
@@ -18,22 +19,19 @@ public class LogReplicationContext {
     private LogReplicationConfig config;
 
     @Getter
+    @Setter
     private TopologyDescriptor topology;
 
     @Getter
     private String localCorfuEndpoint;
 
-    @Getter
-    private IChannelContext channelContext;
-
     /**
      * Constructor
      **/
-    public LogReplicationContext(LogReplicationConfig config,  TopologyDescriptor topology, String localCorfuEndpoint,
-                                 IChannelContext channelContext) {
-        this.config = config;
+    public LogReplicationContext(LogReplicationConfig configs,
```

Comment:
why in plural if its a single config?

Reply:
oh yes.. it may have been carried forward from my PoC changes where different configs were used